R/warn_missing_strata.R

```diff
@@ -1,6 +1,6 @@
 #' Warn about missing stratifying variables in a dataset
 #'
-#' @param data the dataset that should contain the strata
+#' @param est_data the dataset that should contain the strata
```

Comment:
I am avoiding all variable names that conflict with base R keywords. It does throw an error `age` not found. That's because it is reading as base R instead of provided data.